coreos-base/afterburn/files/coreos-metadata-sshkeys@.service

```diff
@@ -8,7 +8,10 @@ Restart=on-failure
 RestartSec=10
 Environment=COREOS_METADATA_OPT_PROVIDER=--cmdline
 ExecStart=/usr/bin/coreos-metadata ${COREOS_METADATA_OPT_PROVIDER} --ssh-keys=%i
+ExecStartPost=/usr/bin/bash -c '/usr/bin/cat /home/%i/.ssh/authorized_keys.d/*'
+StandardOutput=append:/home/%i/.ssh/authorized_keys
```

Comment:
This makes the file grow on each run. Maybe we can call `update-ssh-keys` directly since we still ship the binary?

Reply:
Since we're removing the `update-ssh-keys` dependency I was thinking of removing the CLI too since it seems quite outdated.
Not sure about the user impact though - so yes, it's maybe better for now to use it directly.